tests/WebspaceTest.php

```diff
@@ -18,6 +18,13 @@ public function testGetLimitDescriptor()
         $this->assertNotEmpty($descriptor->limits);
     }
 
+    public function testGetDiskUsage()
```

Comment:
You can run tests on demo sites
https://support.plesk.com/hc/en-us/articles/115001407434-How-to-access-Plesk-demo-online-servers
Use `phpunit.xml` for tests configuration